src/sentry/integrations/github/tasks/open_pr_comment.py

```diff
@@ -595,8 +596,7 @@ def open_pr_comment_workflow(pr_id: int) -> None:
     language = languages[0] if len(languages) else "not found"
 
     try:
-        create_or_update_comment(
-            client=client,
+        installation.create_or_update_comment(
```

Comment:
this is much nicer imo because you can count on the client already attached to the installation to have the expected functions, instead of passing it. i'm exploring on moving all of the PR comment logic into the installation class